packages/build/src/plugins_core/blobs_upload/index.ts

```diff
@@ -28,7 +28,7 @@ const coreStep: CoreStepFunction = async function ({
     siteID: SITE_ID,
     deployID: deployId,
     token: NETLIFY_API_TOKEN,
-    apiURL: NETLIFY_API_HOST,
+    apiURL: NETLIFY_API_HOST.startsWith('http') ? NETLIFY_API_HOST : `https://${NETLIFY_API_HOST}`,
```

Comment:
Not sure I understand why we need the check. Are there any cases where we expect this constant to have a protocol?

Reply:
I don't know exactly if we override it with a protocol somewhere and that's the reason I went with the safety route. I think in the CLI we set the `NETLIFY_API_HOST`

Reply:
e.g. used across the org: https://github.com/netlify/run-builds/blob/e5bd376a98042f638926a3f4d2a31355d3185204/lib/config.js#L4

Reply:
I've removed it